### ISSUE TITLE
Starred topic is showing up multiple times

### ISSUE THREAD

**Issue by aashkadave** [mitmedialab/MediaCloud-Web-Tools#1587]
<img width="1216" alt="Screen Shot 2019-06-03 at 2 44 18 PM" src="https://user-images.githubusercontent.com/31943908/58826429-abb5e700-860e-11e9-9e71-8fcfdee40510.png">

I only have one starred topic right now (image attached) but it's showing up as a starred topic five times instead of one.

**Comment by cindyloo** [comment]
I'll have to ask Aashka if she can reproduce as I cannot with either admin or non-admin account
<img width="484" alt="Screen Shot 2019-06-17 at 7 39 16 PM" src="https://user-images.githubusercontent.com/659092/59643702-b0fb5180-9137-11e9-8f56-776e2bc1b98a.png">

**Comment by aashkadave** [comment]
Hi Cindy! I was able to reproduce it by going to another topic (in this case opiods) and clicking on the "star" button more than once. Rather than getting an indication that the favorite-ing went through (ex: the star becoming colored to indicate a favorite), I didn't see anything change. So I clicked the star multiple times and my "favorites" section now has lots of repeats. Screenshot here: 

<img width="840" alt="Screen Shot 2019-06-19 at 6 44 14 AM" src="https://user-images.githubusercontent.com/31943908/59759365-ec903b80-925d-11e9-805d-4260e8bfdb76.png">
